src/components/CardList/CardList.js

```diff
@@ -1,15 +1,14 @@
 import Card from '../Card/Card';
-import { CardContainer } from '../UI/CardContainer.styled';
-import { CardGrid } from '../UI/CardGrid.styled';
 
 export default function CardList({ entries }) {
-	return (
-		<CardGrid>
-			{entries.map(entry => (
-				<CardContainer key={entry.id}>
-					<Card category={entry.category} name={entry.name} address={entry.address} />
-				</CardContainer>
-			))}
-		</CardGrid>
-	);
+	return entries.map(entry => {
+		return (
+			<Card
+				key={entry.id}
+				category={entry.category}
+				name={entry.name}
+				address={entry.address}
+			/>
+		);
+	});
 }
```

Comment:
Is there a reason, why you didn't use a \<ul> and \<li>\'s when mapping over your list of entries? 🤔

Reply:
You are absolutely right, I adjusted it! :)